Multi-org GA content (#5320)
elastic/docs-content: deploy-manage/deploy/elastic-cloud/create-an-organization.md

  - id: cloud-serverless
---

# Sign up

To get started with {{ecloud}}, sign up for an account:

1. Go to the [{{ecloud}} Sign Up](https://cloud.elastic.co/registration?page=docs&placement=docs-body) page.
2. Choose one of the available sign up methods. You can register with your email address and a password, use a Google or Microsoft account, or [subscribe from a Marketplace](../../../deploy-manage/deploy/elastic-cloud/subscribe-from-marketplace.md).

When you first sign up, you create an organization and start with a trial license.

> An organization is the umbrella for a group of {{ecloud}} resources, users, and account settings. Every organization has a unique identifier. Bills are invoiced according to the billing contact and details that you set for your organization. A user can belong to multiple organizations, or be the owner of multiple organizations. For more details on how to manage your organizations, refer to [](/deploy-manage/cloud-organization.md).

Reviewer: ```suggestion
An organization is the umbrella for a group of {{ecloud}} resources, users, and account settings. Every organization has a unique identifier. Bills are invoiced according to the billing contact and details that you set for your organization. A user can belong to multiple organizations or be the owner of multiple organizations. For more details on how to manage your organizations, refer to [](/deploy-manage/cloud-organization.md).
```

## Trial information [general-sign-up-trial-what-is-included-in-my-trial]

When you create an organization, you start with a free 14-day trial. During this trial, you have access to the following:

% do we want to tell them that they have unlimited trials by creating multiple organizations?

> 

Reviewer 2: Resolving this open question: we don't want to advertise unlimited trials via multiple organizations. Removing the leftover author comment.

```suggestion
```

* [Observability](../../../solutions/observability.md)
* [Security](../../../solutions/security.md)

> When you create a project, you select the project type applicable to your use case, so the most relevant applications and features are surfaced for you from the start.

Reviewer: ```suggestion
When you create a project, you select the project type applicable to your use case, so the most relevant applications and features are shown to you from the start.
```
might be ambiguous for non-native speakers
